golang/x/relay/keeper/querier.go

```diff
@@ -58,24 +89,243 @@ func queryIsAncestor(ctx sdk.Context, path []string, req abci.RequestQuery, keep
 		return []byte{}, types.FromBTCSPVError(types.DefaultCodespace, newDigestErr)
 	}
 
-	// TODO: check that the path is this many items long, error if not
-	limit := uint32(25) // TODO: parse from path, use 240 as default if not in path
+	limit, convErr := strconv.Atoi(path[2]) // TODO: parse from path, use 240 as default if not in path
```

Comment:
use `ParseUint` instead

Reply:
Fixed, changed all instances of this to use ParseUint